decidim-core/app/queries/decidim/highlighted_participatory_processes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module Decidim
+  # This query adds some scopes so the processes are ready to be showed in a
+  # public view.
+  class HighlightedParticipatoryProcesses < Rectify::Query
+    def query
+      Decidim::ParticipatoryProcess.order("promoted DESC").includes(:active_step).order("decidim_participatory_process_steps.end_date ASC").limit(8)
```

Comment:
If I change the order, for some reason, rails doesn't do the join operation and throws a nasty error, si I guess we can't merge them in this specific case.